pages/documentation/simulation/sensitivity/openlf.md

```diff
@@ -74,43 +74,166 @@ We only support for the moment balance type `PROPORTIONAL_TO_GENERATION_P_MAX` a
 
-#### Loss of a single branch
-
-The most frequent contingency is the loss of a single branch.
+#### N-1 case
+Let us start with the most classic case, where a single branch was lost by the network.
  
-Let's introduce $$s_{b,ij,mk}$$ the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ where the branch $$(m,k)$$ represents the outage.
+Let $$s_{b,ij,mk}$$ be the sensitivity of an increase of $$1MW$$ at bus $$b$$ on branch $$(i,j)$$ when the branch $$(m,k)$$ has been disconnected from the network.
 We want to compute this sensitivity.
 
-We call $$b^1$$ the right-hand side vector corresponding with an increase of 1 MW at bus $$b$$. We call $$\theta^1$$ the state vector of voltage angles obtained by solving the equation system on the pre-contingency network that has as right-hand side $$b^1$$.
+Let $$b^1$$ be the right-hand side vector corresponding with an increase of $$1MW$$ at bus $$b$$.
+
+Let $$b^2$$ be the right-hand side vector corresponding with an increase of $$1MW$$ at bus $$m$$ plus a decrease of $$1MW$$ at bus $$k$$.
 
-We call $$b^2$$ be the right-hand side vector corresponding with an increase of 1 MW at bus $$m$$ and a decrease of 1 MW at bus $$k$$. We call $$\theta^2$$ the state vector of voltage angles obtained by solving the equation system on the pre-contingency network that has as right-hand side $$b^2$$.
+Let $$\theta^1$$ be the vector of voltage angles obtained solving the network contraints system on the base network with right-hand side $$b^1$$.
 
-Note that both $$\theta^1$$ and $$\theta^2$$ are built using the same LU decomposition of the constraints matrix $$J$$.
+Let $$\theta^2$$ be the vector of voltage angles obtained solving the network contraints system on the base network with right-hand side $$b^2$$.
+Notice that $$\theta^1$$ and $$\theta^2$$ are built using the same LU decomposition of the constraints matrix $$J$$.
 
-Let $$s_{b,ij}$$ be the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ on the pre-contingency network, we recall that:
+Let $$s_{b,ij}$$ be the sensitivity of an increase of $$1MW$$ at bus $$b$$ on branch $$(i,j)$$ on the base network, we recall that:
 
 $$
 s_{b,ij} = \frac{\theta^1_i-\theta^1_j}{X_{i,j}}
 $$
 
-Let $$s_{mk,ij}$$ be the sensitivity of an increase of 1 MW at bus $$m$$ and a decrease of 1 MW at bus $$k$$, on the pre-contingency network.
-It can be easily computed through the formula:
+Let $$s_{mk,ij}$$ be the sensitivity of an increase of $$1MW$$ at bus $$m$$ plus a decrease of $$1MW$$ at bus $$k$$, on the base network.
+This can be easily computed with the formula:
 
 $$
 s_{mk,ij} = \frac{\theta^2_i-\theta^2_j}{X_{i,j}}
 $$
 
-Then, The post-contingency sensitivity $$s_{b,ij,mk}$$ satisfies:
+Then, The post contingency sensitivity $$s_{b,ij,mk}$$ satisfied:
 
 $$
 s_{b,ij,mk} = s_{b,ij} + \frac{\theta^1_m-\theta^1_k}{X_{m,k} - (\theta^2_m-\theta^2_k)}s_{mk,ij}
 $$
 
+#### N-k case
```

Comment:
Indeed can you removed all the changes before these paragraph or explain me why we have to change what we have committed before ?